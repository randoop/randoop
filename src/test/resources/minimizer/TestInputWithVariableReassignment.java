import org.junit.FixMethodOrder;
import org.junit.Test;
import org.junit.runners.MethodSorters;

@FixMethodOrder(MethodSorters.NAME_ASCENDING)
public class TestInputWithVariableReassignment {

  // Test input with variable reassignment.
  @Test
  public void test1() throws Throwable {
    int j;
    int i = 3;
    j = i++;
    i = 7;

    // Assertion succeeds.
    org.junit.Assert.assertTrue(i == 7);
  }
}
