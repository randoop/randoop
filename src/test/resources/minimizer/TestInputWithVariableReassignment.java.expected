import org.junit.FixMethodOrder;
import org.junit.Test;
import org.junit.runners.MethodSorters;

@FixMethodOrder(MethodSorters.NAME_ASCENDING)
public class TestInputWithVariableReassignmentMinimized {

    // Test input with variable reassignment.
    @Test
    public void test1() throws Throwable {
    }
}
